content/articles/how-to-set-up-rust-api-servers-with-the-diseal-orm-and-the-postgresql/index.md

```diff
@@ -0,0 +1,452 @@
+ORM (Object Relation Mapping) is a common technique for storing, retrieving, updating, and deleting data from a database, specifically within a relational database. Common languages such as Node.js have different ORM libraries that help you achieve the general application of ORM when connecting the database and the application. The application wants to access the data from a database. However, it cannot do that directly. An ORM helps your application connect to a database. The significant advantage is that an ORM helps you create data schemas and relations within your application. Whenever you need to change a specific database field, you only so it in your application with just a few lines of codes to change. This helps you avoid the hectic work of recreating your databases every time to match your new data registry.
+
+The concept of ORM is widely supported by many languages such as Rust. Rust uses the Diseal framework to help you write your schema queries within your rust application. In this guide, we learn more about the Rust Diseal and finally set up a Rust API that uses Diseal to communicate with a PostgreSQL relational database.
```

Comment:
Many Languages should be more than one, not just just a single language as mentioned.